### PR TITLE
Guest post on next-gen image formats

### PR DESCRIPTION
**Pull request by acconrad**

### CODE REVIEW

File: src/jekyll/_posts/2018-05-10-choosing-the-right-next-gen-format-for-your-images.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+This is useful for applications like film and medical imaging because a great deal of data needs to be associated with the image. For example, film stills need to store information about high dynamic range and different color spaces. Medical imaging requires absolute precision in image clarity in spite of the large volume of data. With JPEG 2000, you can encode these files completely losslessly, with good-enough compression performance to make transfer acceptable.
+
+Even if you aren't in the film or healthcare industry, you can take advantage of these new compression techniques in your current JPGs by switching to JPEG 2000. There is one caveat: if you are writing for the web, JPEG 2000 is only [supported on Safari](https://caniuse.com/#feat=jpeg2000), so it will only benefit your customers on Macs, iPhones, and iPads. As long as you use the other two formats that follow, you should definitely consider JPEG 2000 in your arsenal of image formats.
```
Comment (jayroh) [inline src/jekyll/_posts/2018-05-10-choosing-the-right-next-gen-format-for-your-images.md]: What are the benefits to using JPEG 2000 outside of film/healthcare. Are the raw compression gains compelling?

File: src/jekyll/_posts/2018-05-10-choosing-the-right-next-gen-format-for-your-images.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+### JPEG XR
+
+[JPEG XR](https://jpeg.org/jpegxr/index.html) is yet another next-gen JPEG format and a close cousin of JPEG 2000. JPEG XRs application is less intense than JPEG 2000 and was designed in many ways to be the successor of the standard JPEG format.
+
+The main benefits of JPEG XR over the standard format are for enhanced support for HDR imagery. While the original format uses 3 color channels in a bit-depth of 8, JPEG XR supports multiple color formats like monochrome and CMYK and supports bit-depths of 16 or greater. It was also designed from the ground up to support better compression techniques. In other words, it is like JPEG on steroids.
```
Comment (jayroh) [inline src/jekyll/_posts/2018-05-10-choosing-the-right-next-gen-format-for-your-images.md]: 👏 👏 👏 👏 👏 👏 👏 
👍 EXCELLENT STUFF 👍 
👏 👏 👏 👏 👏 👏 👏

File: src/jekyll/_posts/2018-05-10-choosing-the-right-next-gen-format-for-your-images.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+While this is the more practical, versatile file format in comparison to JPEG 2000, it suffers from the same adoption blunders by being available on only a limited set of browsers. As of this writing, JPEG XR is only [supported on IE](https://caniuse.com/#feat=jpegxr), so it will only benefit your customers on Windows machines running some form of IE.
```
Comment (jayroh) [inline src/jekyll/_posts/2018-05-10-choosing-the-right-next-gen-format-for-your-images.md]: > ~~While this is the more practical, versatile file format in comparison to JPEG 2000, it suffers~~
> While this is a more practical and versatile file format compared to JPEG 2000, it suffers

File: src/jekyll/_posts/2018-05-10-choosing-the-right-next-gen-format-for-your-images.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+Combined, these two formats will cover all of your photography needs across the majority of devices. But what about PNGs? And what about open source browsers like Firefox and Chrome?
+
+### WebP
+
+[WebP](https://developers.google.com/speed/webp/) is the open source answer to the next generation of both JPGs and PNGs. The primary application of WebP is the web and is routinely 25% smaller than both optimized PNGs and JPGs.
+
+Because of its web roots, WebP is immediately available in both Chrome and Opera. In addition, WebP has animation support and can be a great replacement for GIFs as well. [`cwebp`](https://developers.google.com/speed/webp/docs/precompiled) is a super easy command line tool to quickly convert your JPGs and PNGs into WebP images, which makes it even easier to convert into than JPEG 2000 or JPEG XR. Luckily, Shubox automates this as well and will let you convert your images into the WebP format when you upload them to S3. How cool is that?
```
Comment (jayroh) [inline src/jekyll/_posts/2018-05-10-choosing-the-right-next-gen-format-for-your-images.md]: PRETTY F'N DOPE IF I DO SAY SO

File: src/jekyll/_posts/2018-05-10-choosing-the-right-next-gen-format-for-your-images.md (hunk 1 above)
Comment (acconrad) [reply src/jekyll/_posts/2018-05-10-choosing-the-right-next-gen-format-for-your-images.md]: as I pointed out in the post, you need it because there is no support for the other next-gen formats in Safari

**Comment by acconrad** [comment]
@jayroh sure i'll get in something like that tonight! thank you for the careful look at this post!

File: src/jekyll/_posts/2018-05-10-choosing-the-right-next-gen-format-for-your-images.md (hunk 1 above)
Comment (jayroh) [reply src/jekyll/_posts/2018-05-10-choosing-the-right-next-gen-format-for-your-images.md]: I get that part -- it's supported by Safari.  But, if your images don't need all that extra metadata support and they're just "dumb" images, are the associated compression gains significant enough to convince someone to use this as their format of choice?

TLDR: Apples to apples, Is this better than the same vanilla JPEG version? By how much?

**Comment by jayroh** [comment]
ONE more thing - rebase onto master and this will be visible as a review "site" on netlify :)

**Comment by acconrad** [comment]
@jayroh changes made! any chance you can rebase? i'm having issues with it locally

**Comment by jayroh** [comment]
Merged from other pr ( #17 )
